contracts/Campaign.sol

```diff
@@ -124,20 +164,77 @@ contract Campaign {
 
         let CampaignTestContract = new web3.eth.Contract(contractABI, campaignAddr);
 
-        CampaignTestContract.getPastEvents("donationMade", options, (error, events) => {
+        CampaignTestContract.getPastEvents("DonationMade", options, (error, events) => {
         if (error) {
             console.log(error);
         } else {
             //for each event, parse out sender and value
             //sender.transfer(value)
-            //emit hasRefunded(sender, value);
+            //emit HasRefunded(sender, value);
         }
         */
     }
 
+    /**
+     * @notice Allows the CampaignFactory contract to transfer any unclaimed donations to itself in the
+     * event that campaign or managing beneficiary is deemed to be untrustworthy
+     * @dev The time to call this function is left to the discretion of the CampaignFactory contract
+     */
     function returnRemainingBalance() public campaignFactoryOnly distrustOnly {
         uint256 remainingBalance =  address(this).balance;
         campaignFactory.transfer(remainingBalance);
-        emit hasReturnedBalance(address(this), remainingBalance);
+        emit HasReturnedBalance(address(this), remainingBalance);
+    }
+
+    /**
+     * @notice Checks if the managing beneficiary has the 'Verified' status
+     * @dev Data is based on the IAM contract
+     * @return true if the beneficiary has the 'Verified' status, false otherwise
+     */
+    function isVerifiedOwner() public view returns (bool) {
+        return IAMContract.isVerified(owner);
+    }
```

Comment:
I think from this method onwards don't need docs since they're `views` and the code mostly speaks for itself. Generally docs/comments are for supplementing what code cannot describe clearly.

Reply:
Agreed. Documentations have been removed in 5ae3edc